src/modules/user/entities/user.entity.ts

```diff
@@ -16,47 +16,35 @@ export enum UserType {
 export class User extends AbstractBaseEntity {
```

Comment:
Are you sure you have the updated code from dev?

Reply:
yeah i sync the dev branch and the one i am working on. May i know why please